core/src/main/resources/jenkins/diagnostics/URICheckEncodingMonitor/message.jelly

```diff
@@ -0,0 +1,16 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <j:if test="${it.checkEnabled}">
+        <script>
```

Comment:
I am not sure you really need JavaScript and AJAX here. It will unlikely work properly since the administrative monitor needs to be calculated before the message is shown. Or not?